tests/tensorflow2/test_estimator.py

```diff
@@ -31,7 +31,7 @@ def test_estimator(out_dir, tf_eager_mode, saveall):
     tnames = trial.tensor_names()
     assert len(trial.steps()) > 0
     if saveall:
-        assert len(tnames) == 301
+        assert len(tnames) >= 301
```

Comment:
Comment.

Reply:
Change the assert statements to something more meaningful and add comments explaining what 301 means